test/integration/requests.test.ts

```
@@ -771,7 +795,7 @@ describe("/requests Task", function () {
     });
 
     it("should not create a new task request if issue does not exist", function (done) {
-      let taskRequestObj = validTaskCreqtionRequest
+      let taskRequestObj = validTaskCreqtionRequest;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'taskRequestObj' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)